lib/liquid/standardfilters.rb

```diff
@@ -128,46 +262,101 @@ def truncatewords(input, words = 15, truncate_string = "...")
       wordlist.join(" ").concat(truncate_string.to_s)
     end
 
-    # Split input string into an array of substrings separated by given pattern.
-    #
-    # Example:
-    #   <div class="summary">{{ post | split '//' | first }}</div>
-    #
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category string
+    # @liquid_name split
+    # @liquid_summary
+    #   Splits a string into an array of substrings based on a given separator.
+    # @liquid_syntax string | split: string
+    # @liquid_return [array[string]]
     def split(input, pattern)
       input.to_s.split(pattern.to_s)
     end
 
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category string
+    # @liquid_name strip
+    # @liquid_summary
+    #   Strips all whitespace from the left and right of a string.
+    # @liquid_syntax string | strip
+    # @liquid_return [string]
     def strip(input)
       input.to_s.strip
     end
 
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category string
+    # @liquid_name lstrip
+    # @liquid_summary
+    #   Strips all whitespace from the left of a string.
+    # @liquid_syntax string | lstrip
+    # @liquid_return [string]
     def lstrip(input)
       input.to_s.lstrip
     end
 
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category string
+    # @liquid_name rstrip
+    # @liquid_summary
+    #   Strips all whitespace from the right of a string.
+    # @liquid_syntax string | rstrip
+    # @liquid_return [string]
     def rstrip(input)
       input.to_s.rstrip
     end
 
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category string
+    # @liquid_name strip_html
+    # @liquid_summary
+    #   Strips all HTML tags from a string.
+    # @liquid_syntax string | strip_html
+    # @liquid_return [string]
     def strip_html(input)
       empty  = ''
       result = input.to_s.gsub(STRIP_HTML_BLOCKS, empty)
       result.gsub!(STRIP_HTML_TAGS, empty)
       result
     end
 
-    # Remove all newlines from the string
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category string
+    # @liquid_name strip_newlines
+    # @liquid_summary
+    #   Strips all newline characters (line breaks) from a string.
+    # @liquid_syntax string | strip_newlines
+    # @liquid_return [string]
     def strip_newlines(input)
       input.to_s.gsub(/\r?\n/, '')
     end
 
-    # Join elements of the array with certain character between them
+    # @liquid_public_docs
+    # @liquid_type filter
+    # @liquid_category array
+    # @liquid_name join
+    # @liquid_summary
+    #   Combines all the items in an array into a single string, separated by a space.
+    # @liquid_syntax array | join
```

Comment:
missing optional param. bet you put this in the spreadsheet. will stop noting them lol